data/templates/ipsec/swanctl/profile.j2

```diff
@@ -13,6 +13,9 @@
         dpd_timeout = {{ ike.dead_peer_detection.timeout }}
         dpd_delay = {{ ike.dead_peer_detection.interval }}
 {%         endif %}
+{%         if uniqreqids %}
```

Comment:
Any reason to not use the common pattern

```suggestion
{% if uniqreqids is vyos_defined %}
```

Which also tests if the supplied variable is not `None`?